addons/auth_signup/models/res_users.py

```diff
@@ -242,7 +251,7 @@ def create(self, vals_list):
             if users_with_email:
                 try:
                     users_with_email.with_context(create_user=True).action_reset_password()
-                except MailDeliveryException:
```

Comment:
Hey :wave: 
Don't forget to replace `action_reset_password` with `_action_reset_password` when relevant in the back-end. As far as I can tell this is the only place where we actually handle it so just replacing it here might be fine, but feel free to look for other places.

Reply:
> Hey wave Don't forget to replace `action_reset_password` with `_action_reset_password` when relevant in the back-end. (and revert this to catch MailDeliveryException again) As far as I can tell this is the only place where we actually handle it so just replacing it here might be fine, but feel free to look for other places.

Hello @thiryr 
Changes have been done, Kindly review them.
Thanks.